README.md

```diff
@@ -1,12 +1,27 @@
-## LaunchDarkly Sample Xamarin Applications
+# LaunchDarkly sample Xamarin application
 
-To demonstrate basic usage of the LaunchDarkly Xamarin SDK, we've built two applications that both do the same thing: one for Android and one for iOS. The only differences between them are in the platform-specific application startup code; the user interface is implemented in a shared project using Xamarin Forms.
+We've built two simple mobile applications that demonstrate how LaunchDarkly's SDK works.
 
-In both applications, there is a single boolean feature flag whose on or off state appears on the screen. The application listens for `FeatureFlagChanged` events from the SDK, so that if the flag is changed on your LaunchDarkly dashboard, it will be updated promptly on the screen.
+Important: these demos are for the _client-side_ .NET SDK, which is suitable for mobile or desktop applications. For server-side use, visit https://github.com/launchdarkly/hello-dotnet-server.
```

Comment:
Consider making this a regular Markdown link? (I think viewing it on GitHub it's clickable either way.)